hyperspy/signal.py

```diff
@@ -1865,7 +1865,8 @@ def __call__(self, axes_manager=None):
         return np.atleast_1d(
             self.data.__getitem__(axes_manager._getitem_tuple))
 
-    def plot(self, navigator="auto", axes_manager=None, **kwargs):
+    def plot(self, navigator="auto", axes_manager=None, 
+            plot_markers=False, **kwargs):
```

Comment:
`plot_markers` need docstring entry (in the right place... `KWARGS_DOCSTRING` ?).